src/main/java/net/contargo/types/truck/LithuanianLicensePlateHandler.java

```diff
@@ -0,0 +1,45 @@
+package net.contargo.types.truck;
+
+/**
+ * @author  Marius van Herpen - mvanherpen@contargo.net
+ */
+public class LithuanianLicensePlateHandler implements LicensePlateHandler {
```

Comment:
Maybe add some examples of typical and supported license plates as well as a link to the documentation used.